ci/prebuilt/thirdparty

```diff
@@ -0,0 +1 @@
+/thirdparty
```

Comment:
@fredlas @htuch I don't think we meant to add these back? cc @lizan

Reply:
Whoops, I know those snuck into another PR I had ongoing. I guess I missed them here.